journalbeat/plan.sh

```diff
@@ -0,0 +1,56 @@
+pkg_svc_user="root"
+pkg_name="journalbeat"
+pkg_description="Journalbeat is a log shipper from systemd/journald to Logstash/Elasticsearch"
+pkg_origin="core"
+pkg_version="5.5.2"
+pkg_maintainer="smartB Engineering <dev@smartb.eu>"
+pkg_license=('Apache 2')
+pkg_upstream_url="https://github.com/mheese/journalbeat"
+pkg_source="https://github.com/mheese/$pkg_name/archive/v$pkg_version.tar.gz"
+pkg_shasum="3cc431a199226b14e318610ee5cfd08d2f3109dc9850591f73a28219c56753c9"
+pkg_deps=(
+  core/lz4
+  smartb/systemd
```

Comment:
Is there a reason you can't rely on the `core/systemd` package?

Reply:
At the time, my PR against `core/systemd` to enable compression had not been accepted yet. I can switch the dep.